pkg/sink/sink.go

```diff
@@ -190,6 +191,7 @@ func (r Sink) executeInterceptors(t *triggersv1.EventListenerTrigger, in *http.R
 		// Set the next request to be the output of the last response to enable
 		// request chaining.
 		request = &http.Request{
+			Method: http.MethodPost,
```

Comment:
Seems to me this should be `Method: in.Method`

Reply:
I think this would be okay for the first request, but not for additional requests since we would generally expect the next request to include whatever body was returned from the last interceptor. wdyt?

Reply:
yes, if the interceptors, the way they are defined as I see in yaml (i.e. an array executed in order), then yes that implies a chain. 

So yes the 1st interceptor would get everything "as-is" from the initial request (unless altered in yaml spec for interceptor) Apply custom headers/custom method (i.e. if #439) then send to interceptor-1. 

Get interceptor-1 result, capture body, send mutated body + original headers + yaml defined headers + method (if #439).... The original body never gets sent but 1x. The original headers always get sent + custom headers. Original method always sent unless overriden per interceptor yaml config.